pkg/operator/ceph/cluster/mgr/spec.go

```diff
@@ -49,9 +49,10 @@ func (c *Cluster) makeDeployment(mgrConfig *mgrConfig, port int) *extensions.Dep
 			Containers: []v1.Container{
 				c.makeMgrDaemonContainer(mgrConfig, port),
 			},
-			RestartPolicy: v1.RestartPolicyAlways,
-			Volumes:       opspec.PodVolumes(""),
-			HostNetwork:   c.HostNetwork,
+			ServiceAccountName: "rook-ceph-mgr",
```

Comment:
How about moving this name to a constant even though it is currently "only" used here and in tests?

Reply:
Is there a test to check that the ServiceAccount is set to that for the mgrs?

There are no changes for mgr tests but osd tests which verify the ServiceAccount name are changed.